eng/ci/templates/official/jobs/build-artifacts.yml

```
@@ -0,0 +1,76 @@
+jobs:
+
+- job: BuildArtifacts
+  displayName: Build Artifacts
+
+  pool:
+    name: 1es-pool-azfunc
+    image: 1es-windows-2022
+    os: windows
+
+  templateContext:
+    outputParentDirectory: $(Build.ArtifactStagingDirectory)
+    outputs:
+    - output: pipelineArtifact
+      displayName: Publish artifacts
+      path: $(Build.ArtifactStagingDirectory)
+      artifact: drop
+
+  steps:
+  - task: DotNetCoreCLI@2
+    displayName: 'Build'
+    inputs:
+      command: 'build'
+      arguments: -c Release
+      projects: |
+        src\**\Microsoft.NET.Sdk.Functions.MSBuild.csproj
+        src\**\Microsoft.NET.Sdk.Functions.Generator.csproj
+        src\**\Microsoft.NET.Sdk.Functions.csproj
+
+  - template: ci/sign-files.yml@eng
+    parameters:
+      displayName: Sign Functions assemblies
+      folderPath: src/Microsoft.NET.Sdk.Functions.MSBuild/bin/Release/
+      pattern: Microsoft.NET.Sdk.Functions.MSBuild.dll
+      signType: dll-strong-name      
+
+  - template: ci/sign-files.yml@eng
+    parameters:
+      displayName: Sign Functions assemblies
+      folderPath: src/Microsoft.NET.Sdk.Functions.Generator/bin/Release/
+      pattern: Microsoft.NET.Sdk.Functions.Generator.dll
+      signType: dll-strong-name      
+
+  - template: ci/sign-files.yml@eng
+    parameters:
+      displayName: Sign third party assemblies
+      folderPath: src/Microsoft.NET.Sdk.Functions.Generator/bin/Release/
+      pattern: Newtonsoft.Json.dll
+      signType: dll
+
+  - template: ci/sign-files.yml@eng
+    parameters:
+      displayName: Sign third party assemblies
+      folderPath: src/Microsoft.NET.Sdk.Functions.Generator/bin/Release
+      pattern: Mono.Cecil.dll
+      signType: dll
+
+  - task: DotNetCoreCLI@2
+    displayName: 'Pack Microsoft.Net.Sdk.Functions package'
+    inputs:
+      command: 'custom'
+      custom: 'pack'
+      arguments: '--no-build -c Release -o artifacts'
```

You have your outputs (above) declared as `$(Build.ArtifactStagingDirectory)`. But you are publishing to `artifacts`. This will lead to no packages published. I recommend:

```suggestion
      arguments: '--no-build -c Release -o $(Build.ArtifactStagingDirectory)/pkg'
```